<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkPage</class>
 <widget class="QWidget" name="NetworkPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:white;
border:1px solid black;
font:12px black &quot;SimHei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="labelCCU1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>CCU1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCCU2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>36</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>CCU2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHMI1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>62</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>HMI1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTBCU1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>88</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TBCU1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRIOM1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>114</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRIOM2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Swich</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPIS1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>PIS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU10">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>200</y>
     <width>53</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU9">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>120</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHVAC1">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>200</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>HVAC1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelACU">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>120</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ACU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU8">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>230</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU6">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>120</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU7">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>200</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU5">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>90</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHVAC2">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>200</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>HVAC2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU12">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>56</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDCU11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>56</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EDCU11</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRIOM5">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelERM1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ERM1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTBCU2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>88</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TBCU2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelERM2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>36</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ERM2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHMI2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>62</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>HMI2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>290</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Switch</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRIOM6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>114</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPIS2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>260</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>PIS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelATC">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>200</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ATC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBCU">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>BCU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Switch</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitch3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Switch</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRIOM3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDCDC1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>DCDC1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRIOM4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMVB">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>5</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51,153,255);
font:16px ;
color:blue;
border:0px;</string>
   </property>
   <property name="text">
    <string>MVB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelETH">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51,153,255);
font:16px ;
color: rgb(255, 0, 255);
border:0px;
</string>
   </property>
   <property name="text">
    <string>Ethernet</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCAN">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51,153,255);
font:16px ;
color:black;
border:0px;</string>
   </property>
   <property name="text">
    <string>CANOpen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDCDC2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>65</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>DCDC2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
